mycroft/skills/intent_service.py

```diff
@@ -153,7 +152,7 @@ def get_skill_name(self, skill_id):
     def reset_converse(self, message):
         """Let skills know there was a problem with speech recognition"""
         lang = _get_message_lang(message)
-        set_active_lang(lang)
+        set_default_lf_lang(lang)
```

Comment:
forward-thinking Gez thinks forward and I approve

Reply:
It was also to avoid people thinking this would set their default language globally across all of Mycroft.